backend/app/src/main/java/com/moh/phlat/backend/esb/json/MaintainHdsRequest.java

```diff
@@ -73,30 +73,102 @@ private Map<String,Object> createEsbMetadata() {
 	
 	private ProviderDetails createProviderDetails() {
 		ProviderDetails providerDetails = new ProviderDetails();
-		providerDetails.setProviderType(data.getHdsType());
+		providerDetails.setOrgNames(createOrgNameDtos());
+		providerDetails.setType("HDS");
+		providerDetails.setProviderType("ORG");
 		providerDetails.setHdsType(createHdsTypeDto());
+		providerDetails.setIdentifiers(createIdentifierDtos());
+		providerDetails.setJurisdiction(createJurisdictionDto());
+		providerDetails.setStatuses(createStatusDtos());
 		providerDetails.setAddresses(createAddressDtos());
```

Comment:
For the following ORG blocks (not mandatory) the mapping is missing . Is is not needed as part of PHLAT? notes
providerRelationships
facilityRelationships

Reply:
The relationships are going to be implemented next after this branch is done and merged, and I don't think we're going to be loading notes through PHLAT